lib/timer.rb

```diff
@@ -0,0 +1,23 @@
+require 'time'
```

Comment:
Good job with creating separate timer class rather than trying to fit it inside the game class.

Reply:
thanks!